syntax = "proto3";

import "fetch_teams/oef_core_protocol/query.proto";

message Update {
    message Uri {
        string agent = 1;
        repeated string oef_core = 2;
    }

    message BulkUpdate {
        repeated Update list = 1;
    }

    Uri uri = 1;
    fetch.oef.pb.Query.DataModel data_model = 2;
}